source/gameengine/Rasterizer/RAS_BucketManager.cpp

```diff
@@ -332,6 +332,10 @@ RAS_MaterialBucket *RAS_BucketManager::FindBucket(RAS_IPolyMaterial *material, b
 		else
 			m_buckets[useinstancing ? SOLID_SHADOW_INSTANCING_BUCKET : SOLID_SHADOW_BUCKET].push_back(bucket);
 	}
+	if (material->IsText()) {
+		m_buckets[TEXT_BUCKET].push_back(bucket);
```

Comment:
In case of libload there will be two material bucket for text material, this is not dangerous.